### PR TITLE
Add SIG Node themes for 1.8

### PR DESCRIPTION
**Pull request by calebamiles**
Attempts to add SIG Node development themes for 1.8. Is this what you had in mind, @derekwaynecarr and @dchen1107?

### CODE REVIEW

File: release-1.8/release_notes_draft.md [hunk 1]
```diff
@@ -50,6 +50,20 @@ promotion indicates RBAC's production readiness and adoption. Today, the
 authorizer is turned on by default by many distributions of Kubernetes, and is a
 fundamental aspect of a secure cluster.
 
+### SIG Node
+
+[SIG Node][] is responsible for the components which support the controlled 
+interactions between pods and host resources as well as managing the lifecycle
+of pods scheduled on a node. For the 1.8 release SIG node was focused on
```
Comment (derekwaynecarr) [inline release-1.8/release_notes_draft.md]: i would just say the following:

sig-node iterated on its ability to run more types of workloads optimally, while making incremental improvements to overall node reliability

File: release-1.8/release_notes_draft.md [hunk 2]
```diff
@@ -50,6 +50,19 @@ promotion indicates RBAC's production readiness and adoption. Today, the
 authorizer is turned on by default by many distributions of Kubernetes, and is a
 fundamental aspect of a secure cluster.
 
+### SIG Node
+
+[SIG Node][] is responsible for the components which support the controlled 
+interactions between pods and host resources as well as managing the lifecycle
+of pods scheduled on a node. For the 1.8 release SIG Node continued to focus
+on supporting the broadest set of workload types, including hardware sensitive
```
Comment (derekwaynecarr) [inline release-1.8/release_notes_draft.md]: s/hardware sensitive/performance sensitive - there is a blog coming out that uses that language.

Comment (derekwaynecarr) [reply release-1.8/release_notes_draft.md]: or "including hardware and performance sensitive workloads"

**Comment by derekwaynecarr** [comment]
LGTM

**Comment by jdumars** [comment]
Huge +1 from me on this

**Comment by Bradamant3** [comment]
So nice! Not sure we can cajole all SIGs into content this good, but it's a great model to work from.

**Pull request merged**